setup.cfg

```
@@ -0,0 +1,22 @@
+[metadata]
+name = sphinxcontrib-towncrier
+url = https://github.com/sphinx-contrib/towncrier
+description = An RST directive for injecting a Towncrier-generated changelog draft containing fragments for the unreleased (next) project version
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+author = Sviatoslav Sydorenko
+author_email = wk+github-open-source@sydorenko.org.ua
+license = BSD 3-Clause License
+license_file = LICENSE
+classifiers =
+    License :: OSI Approved :: BSD License
+
+    Programming Language :: Python :: 3
+
+[options]
+package_dir =
+    = src
+packages = find_namespace:
+
+[options.packages.find]
+where=src
```

let's keep a good spacing style
```suggestion
where = src
```